### ISSUE TITLE
Unable to parse date : Request failed with status code 400

### ISSUE THREAD

**Issue by sheeran-dev** [reportportal/reportportal#2423]
**Describe the bug**
Not able to parse the date. 
I am launching tests using a qavajs  
`npx cross-env ENV=dev qavajs run --config config.ts`

The tests reach report portal and start to be logged in -they remain in progress and in the cli, we can see the following stack trace
```
Error: Request failed with status code 400: {"errorCode":4001,"message":"Incorrect Request. I/O error while reading input message; nested exception is java.io.IOException: Unable to parse date: 2024-11-22T10:05:31.600679Z00"}
URL: <Myurl>/api/v1/rpt_dev_test/item/449d9974-bc9a-45ef-8539-84cb565332bd](https://github.com/reportportal/reportportal/issues/api/v1/rpt_dev_test/item/449d9974-bc9a-45ef-8539-84cb565332bd)
method: PUT
```

**Expected behavior**
No error when sending the reports to reportportal and successful logging on the tests 

**Screenshots**
If applicable

**Versions:**
 - OS, Browser : hosted on a kube - using chrome browser
 - Version of RP : 24.2.0
 - 
**Other information **

pretty similar to this issue https://github.com/reportportal/reportportal/issues/1868

**Comment by Pink-Bumblebee** [comment]
@sheeran-dev , please try to update service-api version to 5.12.1

**Comment by sheeran-dev** [comment]
@Pink-Bumblebee, thanks for your quick answer, we will do so and keep you posted asap !

**Comment by sheeran-dev** [comment]
Hello @Pink-Bumblebee , service-api was successfully updated to 5.12.1 unfortunately we have the same issue : 

```
Error: Request failed with status code 400: {"errorCode":4001,"message":"Incorrect Request. I/O error while reading input message; nested exception is java.io.IOException: Unable to parse date: 2024-11-22T13:57:06.125700Z00"}
URL: <myurl>/api/v1/rpt_dev_test/item/733c217e-d067-4328-a6f7-d3f1f0576c42
method: PUT
```

**Comment by sheeran-dev** [comment]
Hello @Pink-Bumblebee , 
The issue was fixed using upgrading qavajs version from v0.15.0 to v1.0.0  and adding this parameter  "legacyTimeFormat: true" see : 

- https://github.com/qavajs/format-report-portal 


Thanks !
